src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Constants/ConditionalProviderConstants.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration.Constants
+{
+    internal class ConditionalProviderConstants
+    {
+        public const string DisableProviderEnvironmentVariable = "AZURE_APP_CONFIGURATION_PROVIDER_DISABLED";
```

Comment:
Ok makes sense, I wasn't sure between the two options since we also have a file just for the HttpStatusCode constant that we use once too. I think I should just remove that file then too, what do you think?

Reply:
You're right, that couldve been a private constant in the class where its used. However, I think we added a new file for that because HttpStatusCode is possibly extendible if we want to add more error codes in future. But this new environment variable doesn't fall in any category where we might need to add more constants in future. Lets keep the HttpStatusCode for now and remove this one.

Reply:
Oh that's fair, this situation is definitely different. Thanks!